package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lyle",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "generate-docs": "apidoc -i src -o docs",
+    "lint": "eslint \"src/**/*.js\"",
+    "start": "node src/server.js",
+    "start-dev": "nodemon"
+  },
+  "devDependencies": {
+    "apidoc": "^0.18.0",
+    "eslint": "^6.6.0",
+    "nodemon": "^1.19.4"
+  },
+  "dependencies": {
+    "@google-cloud/firestore": "^2.6.0",
+    "@hapi/joi": "^16.1.8",
+    "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
+    "express": "^4.17.1",
+    "googleapis": "^45.0.0",
+    "lodash": "^4.17.15",
+    "uuid": "^3.3.3"
+  }
```

Comment:
`npm outdated` says there are newer versions for a few of these, shall we upgrade before our first commit?

```
Package  Current  Wanted  Latest  Location  Package Type     Homepage
apidoc    0.18.0  0.18.0  0.19.0  lyle      devDependencies  http://apidocjs.com
eslint     6.6.0   6.7.2   6.7.2  lyle      devDependencies  https://eslint.org
nodemon   1.19.4  1.19.4   2.0.1  lyle      devDependencies  http://nodemon.io
```